cmd/clean.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"app/features/cleaner"
```

Comment:
All references to `app/` should be changed at this point to `github.com/Dewberry/v2r`